src/fidesops/service/connectors/saas_query_config.py

```diff
@@ -219,20 +307,53 @@ def generate_update_stmt(
 
         # mask row values
         update_value_map: Dict[str, Any] = self.update_value_map(row, policy, request)
-        update_values: Dict[str, Any] = unflatten_dict(update_value_map)
+        masked_object: Dict[str, Any] = unflatten_dict(update_value_map)
+
+        # map of all values including those not being masked/updated
+        all_value_map: Dict[str, Any] = self.all_value_map(row)
+        complete_object: Dict[str, Any] = unflatten_dict(all_value_map)
+        deep_merge(complete_object, masked_object)
 
         # removes outer {} wrapper from body for greater flexibility in custom body config
-        param_values["masked_object_fields"] = json.dumps(update_values)[1:-1]
+        param_values["masked_object_fields"] = json.dumps(masked_object)[1:-1]
+        param_values["all_object_fields"] = json.dumps(complete_object)[1:-1]
```

Comment:
New placeholder variable so user can pick if they want all the object fields or just the masked fields.